docs/sphinx/manual/building_hpx.rst

```diff
@@ -752,70 +748,52 @@ remember to replace ``/usr/local/include/boost-1_53`` with whatever prefix
 you used in your installation.
 
 Build |hpx|, finally
-..................
+....................
 
 .. code-block:: bash
 
    cd /path/to
    git clone https://github.com/STEllAR-GROUP/hpx.git
    mkdir build-hpx && cd build-hpx
 
-To build with Clang 3.2, execute:
-
-.. code-block:: bash
-
-   cmake ../hpx \
-       -DCMAKE_CXX_COMPILER=clang++ \
-       -DBOOST_INCLUDE_DIR=/usr/local/include/boost-1_53 \
-       -DBOOST_LIBRARY_DIR=/usr/local/lib \
-       -DBOOST_SUFFIX=-clang-darwin32-mt-1_53 \
-   make
-
-To build with Clang 3.3 (trunk), execute:
+To build with Clang, execute:
 
 .. code-block:: bash
 
    cmake ../hpx \
        -DCMAKE_CXX_COMPILER=clang++ \
-       -DBOOST_INCLUDE_DIR=/usr/local/include/boost-1_53 \
-       -DBOOST_LIBRARY_DIR=/usr/local/lib \
-       -DBOOST_SUFFIX=-clang-darwin33-mt-1_53 \
-   make
+       -DBOOST_ROOT=/path/to/boost \
+       -DHWLOC_ROOT=/path/to/hwloc
+   make -j
 
 For more detailed information about using |cmake|, please refer its documentation
 and to the section :ref:`building_hpx`.
 
 Alternative installation method of |hpx| on OS X (Mac)
-....................................................
+......................................................
 
 Alternatively, you can install a recent version of gcc as well as all
 required libraries via MacPorts:
 
 #. Install |macports|
 
-#. Install CMake, gcc 4.8, and hwloc:
+#. Install CMake, gcc, hwloc:
 
    .. code-block:: bash
 
-      sudo port install gcc48
-      sudo port install hwloc
+      sudo brew install cmake
+      sudo brew install boost
+      sudo brew install hwloc
+      sudo brew install make
 
-   You may also want:
+#. You may also want:
 
    .. code-block:: bash
 
-      sudo port install cmake
-      sudo port install git-core
+      sudo brew install gperftools
 
-#. Make this version of gcc your default compiler:
-
-   .. code-block:: bash
-
-      sudo port install gcc_select
-      sudo port select gcc mp-gcc48
-
-#. Build Boost manually (the Boost package of MacPorts is built with Clang, and
-   unfortunately doesn't work with a GCC-build version of |hpx|):
+#. If you need to build Boost manually (the Boost package of MacPorts is built
+   with Clang, and unfortunately doesn't work with a GCC-build version of |hpx|):
```

Comment:
I think for Mac we need to enable using Boost context (`-DHPX_WITH_GENERIC_CONTEXT_COROUTINES=On`)